### ISSUE TITLE
self-supervision prediction

### ISSUE THREAD

**Issue by yeqiyuan** [google-research/vision_transformer#66]
Hello, sir, do you have the link of self-supervision prediction experiments.(using mask to represent patches). Could you share it.

**Comment by lucasb-eyer** [comment]
No, we do not plan to publish this code. But it should be relatively straightforward for you to implement a variant of this :)

**Comment by yeqiyuan** [comment]
Thank you.
